admin/src/routes/projects/CreateWorkflow.tsx

```diff
@@ -61,52 +58,53 @@ const CreateWorkflow: Component = () => {
   };
 
   return (
-    <div class="flex gap-x-12">
-      <div class="w-48" />
-
-      <div class="flex-1 flex flex-col">
-        <div class="max-w-screen-md space-y-2">
-          <Heading size={3}>Objective</Heading>
-
-          <Paragraph size="sm">
-            What do you want to extract from the web? You may state this in
-            plain English. Feel free to use topics and keywords that you care
-            about. Pixlie will continue crawling the web as long as pages match
-            your objective.
-          </Paragraph>
-
-          <TextArea
-            id="projectObjective"
-            name="objective"
-            isEditable
-            onChange={handleChange}
-            value={formData().objective}
-          />
-
-          {/* <div class="flex items-center gap-x-2">
-            <Toggle />
-            Specify links to start crawling from.
-          </div>
-
-          <Heading size={3}>Starting links</Heading>
-          <NodeGrid nodeType={"Link"} source={} />
-          <div class="max-w-screen-sm">
-            <LinkForm />
-          </div> */}
-
-          <div class="pt-6 flex space-x-3">
-            <div class="flex-1" />
-            <Button label="Cancel" href="/" />
-            <Button
-              label="Save"
-              colorTheme="success"
-              onClick={handleFormSubmit}
-            />
-          </div>
+    <div
+      class="flex w-full h-full justify-center items-end grid grid-cols-3 grid-rows-3 relative"
+      style={{
+        "background-image":
+          "url('https://pixlie.com/_astro/hero-image.DdgBYhys_2wzNdY.webp')",
```

Comment:
Ideally, we should add all images to the repos assets, instead of loading them from other sources.

Also, I feel we should remove the background or make it very subtle, maybe having ~0.25 opacity. It is quite hindering to the visibility of the interface to enter the objective, IMHO.